pkg/export/export.go

```diff
@@ -170,6 +170,12 @@ type ExporterOpts struct {
 	Location  string
 	Cluster   string
 
+	// If true, automatically populate the project id in an exemplar labelset.
+	PopulateExemplarProjectID bool
```

Comment:
I would say let's avoid bools flag. They are not flexible and scalable. What about empty `ExemplarProjectID` stopping this to populate that extra label?

Reply:
I think the semantics of this would get confusing since this is supposed to be both overridable and disableable. 

An empty `ExemplarProjectID` seems to make sense as the case where you would not autopopulate the `project_id`, but not passing something in seems to imply that it is the default state which is not what we want. We want the default state to be to autopopulate the exemplar projectID. 

Unless you mean empty as in the literal empty string (`""`) which also seems to be semantically confusing.

I'll defer to you, @pintohutch and @lyanco  on this one.